features/step_definitions/login_steps.rb

```diff
@@ -1,14 +1,21 @@
 Given(/^the login page$/) do
-  #put your code here
-  pending
+  visit '/'
 end
 
 When(/^I log in with proper credentials$/) do
-  #put your code here
-  pending
+  within("#session") do
+    fill_in 'Username', :with => 'cchoi'
+    fill_in 'Password', :with => 'cchoi'
+  end
+  click_button 'Login'
 end
 
 Then(/^I should see the secret page$/) do
-  #put your code here
-  pending
+  expect(current_path).to eq '/dashboard'
+end
+
+And(/^I should see the content "(.*?)"$/) do |text|
+  within("#session") do
+    page.should have_content text
```

Comment:
You have a mix of should & expect syntax in your step defs. Expect is preferred.
http://betterspecs.org/#expect